charts/hono/templates/_helpers.tpl

```diff
@@ -16,7 +16,8 @@
 Expand the name of the chart.
```

Comment:
Would be good to add this here:
```
The scope passed in is expected to be a dict with keys
- (mandatory) "dot": the root (".") scope
```

Also applies to the documentation sections for:
- `define "hono.fullname"`
- `define "hono.chart"`
- `define "hono.std.labels`

Reply:
done